Dockerfile

```diff
@@ -1,5 +1,24 @@
-FROM ruby:2.5.1
-RUN apt-get update -qq && apt-get install -y ruby-dev
+FROM alpine:3.8
+
+LABEL org.label-schema.maintainer="Puppet Bolt Team <team-direct-change-bolt@puppet.com>" \
+      org.label-schema.vendor="Puppet" \
+      org.label-schema.url="https://github.com/puppetlabs/bolt" \
+      org.label-schema.name="PE Bolt Server" \
+      org.label-schema.license="Apache-2.0" \
+      # Is there a way to set this?
+      #org.label-schema.version=$PUPPET_VERSION \
+      org.label-schema.vcs-url="https://github.com/puppetlabs/bolt" \
+      # Same with these
+      #org.label-schema.vcs-ref="b75674e1fbf52f7821f7900ab22a19f1a10cafdb" \
+      #org.label-schema.build-date="2018-05-09T20:10:01Z" \
+      #org.label-schema.schema-version="1.0" \
+      org.label-schema.dockerfile="/Dockerfile"
+
+RUN \
+apk update && apk upgrade && \
+apk --no-cache add build-base ruby-dev git && \
+gem install bundler --no-ri --no-rdoc && \
+bundle config --global silence_root_warning 1
 
 RUN mkdir /bolt-server
 WORKDIR /bolt-server
```

Comment:
I saw a recommendation to use `RUN echo 'gem: --no-document' > /etc/gemrc` to avoid installing gem documentation. Might make sense to slim this down. Same with using `bundle install --no-cache`.